Doc/whatsnew/3.9.rst

```diff
@@ -202,6 +202,16 @@ now raises :exc:`ImportError` instead of :exc:`ValueError` for invalid relative
 import attempts.
 (Contributed by Ngalim Siregar in :issue:`37444`.)
 
+ipaddress
+---------
+
+:mod:`ipaddress` now supports IPv6 Scoped Addresses (IPv6 address with suffix ``%<scope_id>``).
+
+Scoped IPv6 could be parsed to :class:`ipaddress.IPv6Address` constructor.
+If present, scope zone ID is avalaible through :attr:`scope_id` attribute.
```

Comment:
1) Spelling fix "avalaible" -> "available"
2) Added a "the"

```suggestion
If present, scope zone ID is available through the :attr:`scope_id` attribute.
```

Reply:
@aeros  Thank you! After checks pass, I`m going to ask for core review via python-dev@python.org again.